quickwit-doc-mapper/src/query_builder.rs

```diff
@@ -277,5 +278,24 @@ mod test {
             TestExpectation::Err("No default field declared and no field specified in query."),
         )
         .unwrap();
+        check_build_query(
+            "server.running:true",
+            vec![],
+            None,
+            TestExpectation::Ok("TermQuery"),
+        )
+        .unwrap();
+    }
+
+    #[test]
+    #[should_panic(expected = "provided string was not `true` or `false`")]
+    fn test_build_query_not_bool_should_fail() {
+        check_build_query(
+            "server.running:not a bool",
+            vec![],
+            None,
+            TestExpectation::Err("Expected a success when parsing TermQuery, but got error"),
+        )
+        .unwrap();
```

Comment:
I can remove the tests if they are not required

Reply:
I think this kind of test should be made on tantivy.